share/man/man1/intro.1

```diff
@@ -54,24 +68,40 @@ Some commands attempt to describe the nature of the failure by using
 exit codes as defined in
 .Xr sysexits 3 ,
 while others simply set the status to an arbitrary value >0 (typically 1).
+.Sh FILES
+.Bl -tag -width "/usr/local/bin/tab" -compact
+.It Pa /bin/
+Fundamental
+.Bx
+user commands.
+.It Pa /usr/bin/
+General commands included with the base system.
+.It Pa /usr/local/bin/
+Locally installed commands from
+.Xr pkg 8 or
+.Xr ports 7 .
+.El
 .Sh SEE ALSO
 .Xr apropos 1 ,
 .Xr man 1 ,
+.Xr which 1 ,
 .Xr intro 2 ,
 .Xr intro 3 ,
 .Xr sysexits 3 ,
 .Xr intro 4 ,
 .Xr intro 5 ,
 .Xr intro 6 ,
 .Xr intro 7 ,
+.Xr ports 7 ,
 .Xr security 7 ,
 .Xr intro 8 ,
+.Xr pkg 8 ,
 .Xr intro 9
 .Pp
 Tutorials in the
 .%T "UNIX User's Manual Supplementary Documents" .
 .Sh HISTORY
-The
+A
 .Nm
-manual page appeared in
+for section one first appeared in 1975 with
```

Comment:
I think the previous line was much better, I think explicitly specifying the date isn't exactly needed/relevant here. Also, "A intro"?

Maybe?
```diff
- A intro for section one first appeared in 1975 with Version 6 AT&T UNIX.
+ The intro manual page first appeared in Version 6 AT&T UNIX.
```

Reply:
How do you feel about `The intro page for section one appeared in 1975 with Version 6 AT&T UNIX`?

Someone reading this for the first time is likely less than half the age of this page. They've never heard of At v6 and likely uninterested in going to investigate that separare idea. Half a century of this page is a profound fact of this idea, with lots of implications.

Reply:
"for section one" probably is a bit more verbosity, personally...
"The intro manual page first appeared in 1975 with the Version 6 AT&T UNIX"
seems good to me.

Reply:
Adding "manual" and "first" would make it wrap on a standard console.

I thought "the intro manual" reads strangely as there are many intro pages added at different times. "section" is verbose, but ends ambiguity without wrapping.

Thanks!

Reply:
Without the word "manual", it sounds a bit odd, that's why I suggested that... but that's just my view, other may have different tastes :)

for section... I think "intro(1)" can also be a considerable choice, as it's clearly says man 1 intro.

Reply:
"The intro(1) manual page first appeared in Version 6 AT&T Unix."
eg, just insert 'first appeared'. I don't think the year should be included. We don't elsewhere.

Reply:
`The .Nm Ns Pq 1 manual page first appeared in .At v6 .` sounds great and addresses my original concern. I'll force push that (and mark this thread resolved?) Thanks so much for the feedback guys.